draft-irtf-cfrg-vdaf.md

```diff
@@ -260,8 +260,8 @@ informative:
   TestVectors:
     title: "Test vectors for Prio3 and Poplar1"
     target: "https://github.com/cfrg/draft-irtf-cfrg-vdaf/tree/main/test_vec/vdaf"
-    seriesinfo: "commit hash <TODO>"
-    date: September 2024
+    seriesinfo: "commit hash 1891650bb8d23eb82663a0c05a5e37e97c4b5a96"
```

Comment:
FWIW the [style guide](https://www.rfc-editor.org/styleguide/part2/#ref_repo) says to use a short form of the commit hash

Reply:
I saw this but intended to ignore it given the probability of a collision on the short hash. But I suppose the date ensures it's unambiguous, as long as we don't have a collision in the next month's worth of commits :D